integrations/tensorflow/test/tfe2e/math/test_matrix.yaml

```diff
@@ -0,0 +1,244 @@
+lists:
```

Comment:
First off, this directory/math_test.py is a bit of an oddball because one source file expands to 500+ tests -- which makes it of an unusual size :) When I was fiddling with it, it got really unwieldy to put that scale of thing in the source file itself as annotations. I did set it up so that I could also scan the source files in some way for cases of more usual size, thereby keeping things together. I didn't actually finish that yet because I hadn't gotten to one of those examples.

Second, even just splitting this out to explicit generation and a config step that I can run is an improvement (a big one). I went through a bunch of rounds on this case (which, this case is quite complex) and found it to be a fairly improved development experience. Obviously different ways to do it, but we're not bazlifying another thing in this area. Not gonna touch that anymore. I'd rather have a stand alone tool that does what I want, how I want it, and it doesn't need me to keep the state of 3-4 different systems in mind to touch it.

We'll have a script on the google side to pre-generate these things just like XLA does (for similar reasons). Aligning this too closely with the build system is a maxima that you will never break out of and seriously hampers development work.

Happy to get on a f2f to discuss.